File: app/src/main/java/com/imangazalievm/bubbble/di/modules/UserProfilePresenterModule.java
package com.imangazalievm.bubbble.di.modules;

import com.imangazalievm.bubbble.di.scopes.Presenter;

import dagger.Module;
import dagger.Provides;

@Module
public class UserProfilePresenterModule {

    private final long userId;

    public UserProfilePresenterModule(long userId) {
        this.userId = userId;
    }

    @Provides
    @Presenter
    long provideUserId() {
        return userId;
    }

}

File: app/src/main/java/com/imangazalievm/bubbble/di/userprofile/UserProfilePresenterModule.kt
package com.imangazalievm.bubbble.di.userprofile;

import com.imangazalievm.bubbble.di.global.scopes.Presenter;

import dagger.Module;
import dagger.Provides;

@Module
public class UserProfilePresenterModule {

    private final long userId;

    public UserProfilePresenterModule(long userId) {
        this.userId = userId;
    }

    @Provides
    @Presenter
    long provideUserId() {
        return userId;
    }

}
